appserver/payara-appserver-modules/microprofile/jwt-auth/src/main/java/fish/payara/microprofile/jwtauth/jwt/JwtTokenParser.java

```diff
@@ -63,9 +64,19 @@
  * @author Arjan Tijms
  */
 public class JwtTokenParser {
-
+    
+    private final static String DEFAULT_NAMESPACE = "https://payara.fish/mp-jwt/";
+    
     private final List<Claims> requiredClaims = asList(iss, sub, exp, iat, jti, groups);
+    
+    private final boolean enableNamespacedClaims;
+    private final Optional<String> customNamespace;
 
+    public JwtTokenParser(Optional<Boolean> enableNamespacedClaims, Optional<String> customNamespace) {
+        this.enableNamespacedClaims = enableNamespacedClaims.orElse(false);
+        this.customNamespace = customNamespace;
+    }
+    
```

Comment:
```suggestion

    public JwtTokenParser() {
        this(Optional.empty(), Optional.empty());
    }
```

Our TCK runner depends on the default constructor.